deployment_and_ssl_doc.md

```diff
@@ -5,10 +5,38 @@ to Oxleas to an Azure instance that has been Provisioned by SLaM and will assume
 you have both. You will also need to have an AWS SES account set up and ready to
 use.
 
+This documentation is also used as a basis for deploying similar applications,
+which are listed below.
+
+* [Healthlocker](https://github.com/healthlocker/healthlocker/blob/master/deployment_and_ssl.md)
+
+If this document is added to, please ensure the specific instructions in other
+documentation is updated accordingly.
+
+## Before you get started
+In order to go through all the steps in this README, you will need:
+
+* An Azure instance which you have access to
```

Comment:
Strictly speaking, does it have to be an Azure instance?  I don't think anything in here is Azure specific.  It's all Linux VM related.